KinSDK/KinSDK/Core/KinError.swift

```diff
@@ -38,11 +38,21 @@ public enum KinError: Error {
      */
     case balanceQueryFailed (Error)
 
+    /**
+     Querying for the account aggregated balance failed.
+     */
+    case aggregatedBalanceQueryFailed (Error)
```

Comment:
why there's a special error for each query, isn't reusing general errors i.e. AccountNotFound in case of 404, general IO error in case of network error etc.?

Reply:
This is a good solution, but this does not exist now. I'm only adding to the existing design. I'll add to my SDK changes note to improve the errors.